src/Microsoft.DotNet.Build.Tasks/Targets/post-process-assembly.targets

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <IntermediateAssemblyPath>$(IntermediateOutputPath)$(TargetName)$(TargetExt)</IntermediateAssemblyPath>
+    <PostProcessAssemblyMarker>$(IntermediateAssemblyPath).postprocessed</PostProcessAssemblyMarker>
+  </PropertyGroup>
+
+  <Target Name="PostProcessAssembly"
+    AfterTargets="CoreCompile"
+    Inputs="$(IntermediateAssemblyPath)"
+    Outputs="$(PostProcessAssemblyMarker)">
+
+    <CallTarget Targets="@(PostProcessAssemblySteps)" />
```

Comment:
Yes my suggestion, at least for now would be to undo your changes to sign.targets and delete your new post-processing target and just hook the FillParitialFacade target up into TargetsTriggeredByCompilation without inputs/outputs.